plugins/inspect/os_inspector.rb

```diff
@@ -113,4 +118,17 @@ def get_os_from_suse_release(system)
     os.version = result["version"]
     os
   end
+
+  # checks for redhat standard: /etc/redhat-release
+  def get_os_from_redhat_release(system)
+    redhat_release = system.read_file("/etc/redhat-release")
+    return if !redhat_release
+
+    result = Hash.new
+    result["name"], result["version"] = redhat_release.split("\n").first.split(" release ")
+
+    os = Os.for(result["name"])
+    os.version = result["version"]
+    os
+  end
```

Comment:
It would be good to have a separate test for this method. Parsing a file can nicely be encapsulated in a unit test, and then the higher-level methods can more easily be tested without relying on details of the parsing.